content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+When you're making a game in Bevy, you will have a single `App`, which stores and handles all of your game logic and data.
+Let's make a trivial Hello World app.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::build()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+So, we first create our [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html) by telling it to [`build`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.build), performing initialization steps.
+This produces an [`AppBuilder`](https://docs.rs/bevy/latest/bevy/app/struct.AppBuilder.html), which we can use to configure the `App`.
+We do so by add a simple system, which prints "Hello, Bevy!" when it is run.
+Finally once all of our work is complete, we call [`AppBuilder::run()`](https://docs.rs/bevy/latest/bevy/app/struct.AppBuilder.html#method.run) to actually make our app *do things*.
+
+## What makes an App?
+
+So, what sort of data does are `App` really store?
+Turns out: it's fairly simple.
+Looking at the docs for [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html), we find three fields:
+
+```rust
```

Comment:
I would prefer to not immediately throw stuff like `Box<dyn Fn(App) + 'static, Global>` at people on the second page of the Bevy Book. We can introduce World/Schedule/runner concepts without this.

By not inlining the App struct, we also insulate against future changes (such as the infrastructure required to support deferred plugin init).

Reply:
Yes, I also talked with Alice about this. We ended up deciding to simply link to the API docs instead, where the user can also check the engine source code if they want. By the way, `App` fields are private implementation details and therefore should definitely not be in the Book.